crates/diesel_models/src/process_tracker.rs

```diff
@@ -209,6 +209,7 @@ pub enum ProcessTrackerRunner {
     DeleteTokenizeDataWorkflow,
     ApiKeyExpiryWorkflow,
     OutgoingWebhookRetryWorkflow,
+    StripeAttachAccountWorkflow,
```

Comment:
Is this workflow specific to stripe?

Reply:
Yes, specific to Stripe. They have an asynchronous model for onboarding users on StripeConnect